wbooks-api/app/controllers/books_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  include Wor::Paginate
+
+  def index
+    render_paginated Book
+  end
+
+  def show
+    render json: Book.find(book_params)
+  end
+
+  private
+
+  def book_params
+    params.require(:id)
+  end
+
+  def books_list_params
```

Comment:
who uses this method??

Reply:
The Apiary doc of the API says the index route should receive all this params as required ones, I set this method to enable the use of this params, but it is not being used because it is not specified to use them in any way. @icoluccio told me to set them as strong params anyway, I don't know if is better to remove the method